### PR TITLE
Hotfix/deb mongodb req

### PR DESCRIPTION
**Pull request by itaymmguardicore**
# Feature / Fixes
> Fix deb dependency of mongodb

* [X] Have you added an explanation of what your changes do and why you'd like to include them?
* [X] Have you successfully tested your changes locally?

##  Changes
  - Add mongodb installation script
  - Modify & Clean files for creating .deb for both island and mongo

### CODE REVIEW

**Comment by danielguardicore** [comment]
@VakarisZ make sure this doesn't break deployment scripts

File: monkey/monkey_island/linux/install_mongo.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+export os_version_monkey=$(cat /etc/issue)
+MONGODB_DIR=$1 # If using deb, this should be: /var/monkey/monkey_island/bin/mongodb
+
+if  [[ $os_version_monkey == "Ubuntu 14.04"* ]] ;
```
Comment (danielguardicore) [inline monkey/monkey_island/linux/install_mongo.sh]: Do we even support Ubuntu 14.04? The python version there is too old for Tornado which I think we require on the Island.

File: monkey/monkey_island/linux/install_mongo.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1404-3.6.12.tgz"
+elif [[ $os_version_monkey == "Ubuntu 16.04"* ]] ;
+then
+	echo Detected Ubuntu 16.04
+	export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-3.6.12.tgz"
+elif [[ $os_version_monkey == "Ubuntu 18.04"* ]] ;
+then
+	echo Detected Ubuntu 18.04
+	export tgz_url="https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.0.8.tgz"
+elif [[ $os_version_monkey == "Debian GNU/Linux 8"* ]] ;
```
Comment (danielguardicore) [inline monkey/monkey_island/linux/install_mongo.sh]: Don't support debian 8

File: monkey/monkey_island/linux/install_mongo.sh (hunk 1 above)
Comment (itaymmguardicore) [reply monkey/monkey_island/linux/install_mongo.sh]: fixed

**Pull request merged**